evening_session.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'session'
+
+class EveningSession
```

Comment:
Why are we calling super when the class does not have a user-defined super calss